packages/terra-form-field/src/terra-dev-site/test/form-field/DefaultField.test.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Base from 'terra-base';
+import Field from '../../../../lib/Field';
+
+const locale = document.getElementsByTagName('html')[0].getAttribute('lang');
+
+const DefaultField = () => (
+  <Base locale={locale}>
+    <h3> Default Field</h3>
```

Comment:
Remove the space before D

Reply:
[Updated.](https://github.com/cerner/terra-core/pull/1537/commits/89ffbfe150e43b31ee6d3fc6fd022d0610b247d7#diff-f908e7da845ac45c958b8a06974c63c9R8)